src/Attachment.php

```diff
@@ -57,16 +61,14 @@ public function prepareAttachmentForJs(array $response): array
     /**
      * Same as wp_ajax_query_attachments() but with switch_to_blog support.
      *
-     * @since   2015-01-26
      * @return void
+     * @since   2015-01-26
      */
     public function ajaxQueryAttachments()
     {
         // phpcs:disable WordPress.CSRF.NonceVerification.NoNonceVerification
         // phpcs:disable
-        $query = isset($_REQUEST['query'])
-            ? (array)wp_unslash($_REQUEST['query'])
-            : [];
+        $query = isset($_REQUEST['query']) ? (array)$_REQUEST['query'] : array();
```

Comment:
We write the plugin for php >=7.0.0, so please change to the new art of array string `[]`. Thanks!

Reply:
It's done ,Unfortunately tests are not copied when the project is forked by me, this is due to the lack of test files. :+1: